models/DataObject/ClassDefinition/Data/Hotspotimage.php

```diff
@@ -38,14 +38,14 @@ class Hotspotimage extends Model\DataObject\ClassDefinition\Data\Image
     /**
      * Type for the column to query
      *
-     * @var string
+     * @var array
```

Comment:
please fix return doc here as well: https://github.com/pimcore/pimcore/blob/83ece3824eb4666d212e5d2dfdbb0f80d979a07f/models/DataObject/ClassDefinition/Data/Hotspotimage.php#L168
Thanks!

Reply:
What do you mean? I think `DataObject\Data\Hotspotimage|null` is correct for `Hotspotimage::getDataFromResource`

Reply:
ok, you are right. I misread it. 😄